specification/network/resource-manager/Microsoft.Network/stable/2019-08-01/examples/VirtualNetworkGatewayUpdate.json

```diff
@@ -23,6 +23,7 @@
         "gatewayType": "Vpn",
         "vpnType": "RouteBased",
         "enableBgp": false,
+        "enableDnsForwarding": true,
```

Comment:
Please also add new properties in example response (since without them it's not acquirate)